builder/includes/associations.form.inc

```diff
@@ -298,3 +295,151 @@ function xml_form_builder_enable_association($form_name, $id) {
   drupal_set_message(t('Successfully enabled association.'));
   drupal_goto(xml_form_builder_get_associate_form_path($form_name));
 }
+
+/**
+ * Defines the association edit form.
+ *
+ * @param array $form
+ *   The drupal form.
+ * @param array $form_state
+ *   The drupal form state.
+ * @param string $form_name
+ *   The name of an XML form.
+ * @param string $id
+ *   The id of this association.
+ *
+ * @return array
+ *   The drupal form definition.
+ */
+function xml_form_builder_edit_association_form(array $form, array &$form_state, $form_name, $id) {
+  module_load_include('inc', 'xml_form_builder', 'includes/associations');
+  $associations = xml_form_builder_get_associations(array($form_name), array(), array(), FALSE);
```

Comment:
As we have the `$id` can we not call this helper instead: https://github.com/Islandora/islandora_xml_forms/blob/7.x/builder/includes/associations.inc#L21?